test/fixtures/html/components/form/search-global.html

```diff
@@ -1,4 +1,25 @@
-<div class="search__global">
-   <form>
-        <div class="input-group"><label class="sr-only" for="search_box"></label><input id="search_box" maxlength="1000" name="q" pattern="^(?!.*<script|<Script).*$" required="" type="search"><button aria-label="submit search" class="btn--white" type="submit"><svg aria-hidden="true" data-icon="search" height="20" viewBox="0 0 20 20" width="20" xmlns="http://www.w3.org/2000/svg"><title>Search icon</title><g fill="#fff"><path d="M7.5 15C3.4 15 0 11.6 0 7.5 0 3.4 3.4 0 7.5 0 11.6 0 15 3.4 15 7.5 15 11.6 11.6 15 7.5 15ZM7.5 13C10.5 13 13 10.5 13 7.5 13 4.5 10.5 2 7.5 2 4.5 2 2 4.5 2 7.5 2 10.5 4.5 13 7.5 13Z"></path><polygon points="14.8 10.8 16.8 10.8 16.8 20.8 14.8 20.8" transform="translate(15.778175 15.778175)rotate(-45)translate(-15.778175 -15.778175)"></polygon></g></svg></button></div>
-    </form><a aria-label="open search" class="search__global__toggle" href="/search"><svg aria-hidden="true" data-icon="search" height="20" viewBox="0 0 20 20" width="20" xmlns="http://www.w3.org/2000/svg"><title>Search icon</title><g fill="#fff"><path d="M7.5 15C3.4 15 0 11.6 0 7.5 0 3.4 3.4 0 7.5 0 11.6 0 15 3.4 15 7.5 15 11.6 11.6 15 7.5 15ZM7.5 13C10.5 13 13 10.5 13 7.5 13 4.5 10.5 2 7.5 2 4.5 2 2 4.5 2 7.5 2 10.5 4.5 13 7.5 13Z"></path><polygon points="14.8 10.8 16.8 10.8 16.8 20.8 14.8 20.8" transform="translate(15.778175 15.778175)rotate(-45)translate(-15.778175 -15.778175)"></polygon></g></svg><svg aria-hidden="true" data-icon="close" height="20" viewBox="0 0 20 20" width="20" xmlns="http://www.w3.org/2000/svg"><title>Cross</title><g stroke="#fff"><g><line stroke-width="2" x1="0" x2="18" y1="0" y2="18"></line></g><g><line stroke-width="2" x1="0" x2="18" y1="18" y2="0"></line></g></g></svg></a></div>
+<form>
+        <div class="input-group"><label class="sr-only" for="search_box"></label><input id="search_box" maxlength="1000" name="q" pattern="^(?!.*<script|<Script).*$" aria-label="enter a search term" autocomplete="off" required="" type="search"><button aria-label="submit search" class="btn--white" type="submit" data-tracking="search"><svg aria-hidden="true" data-icon="search" height="20" viewBox="0 0 20 20" width="20" xmlns="http://www.w3.org/2000/svg">
+                    <title>Search icon</title>
```

Comment:
Need to remove the data-tracking attribute

Reply:
parliament.uk-pugin templates require updating as I lifted this attribute from the template